src/runtime/logging.cc

```diff
@@ -17,16 +17,15 @@
  * under the License.
  */
 
-#ifdef TVM_BACKTRACE_DISABLED
+#if TVM_USE_LIBBACKTRACE == 0
+#include <dmlc/logging.h>
+
 #include <string>
 
-// TODO(bkimball,tkonolige) This inline function is to work around a linking error I am having when
-// using MSVC If the function definition is in logging.cc then the linker can't find it no matter
-// what kind of attributes (dllexport) I decorate it with. This is temporary and will be addressed
-// when we get backtrace working on Windows.
 namespace tvm {
 namespace runtime {
-__declspec(dllexport) std::string Backtrace() { return ""; }
+// Fallback to the dmlc implementation when backtrace is not available.
+std::string Backtrace() { return dmlc::StackTrace(); }
```

Comment:
I don't think we should fall back to dmlc's stack trace implementation. It uses its own set of `-D` flags to control behavior. Users will be confused by the need to set `-DTVM_USE_LIBBACKTRACE=0` and `-DDMLC_LOG_STACK_TRACE=0`.

Reply:
the dmlc::StackTrace at least gives us something over nothing, so it is useful for systems when the backtrace is not enabled.

Reply:
Can you move it into TVM then, so we have consistent flags.

Reply:
Given the fallback is less frequently used and this PR did fixes the previous problem and bring in new changes, I think this change is OK as it is. We can consider bring the new impl back later if needed.